geode-dunit/src/main/java/org/apache/geode/cache30/MultiVMRegionTestCase.java

```diff
@@ -4972,13 +4972,9 @@ public void run2() throws CacheException {
               // reset slow
               org.apache.geode.internal.cache.InitialImageOperation.slowImageProcessing = 0;
               // if global scope, the region doesn't get destroyed until after region creation
-              try {
-                Thread.sleep(3000);
-              } catch (InterruptedException ie) {
-                fail("interrupted");
-              }
-              assertThat(getRootRegion().getSubregion(name) == null
-                  || getRegionAttributes().getScope().isGlobal()).isTrue();
+              await()
```

Comment:
Couldn't you just do:

```await().until(() -> getRootRegion().getSubregion(name) == null || getRegionAttributes().getScope().isGlobal())```

and eliminate the asserts?  Just a bit less verbose I think, but its up to you.